src/CrystalStacker.xml

```diff
@@ -0,0 +1,21 @@
+<SPDX name="CrystalStacker License" identifier="CrystalStacker" osi-approved="false">
+  <urls>
+    <url>https://fedoraproject.org/wiki/Licensing:CrystalStacker?rd=Licensing/CrystalStacker</url>
+  </urls>
+  <license>
+    <p>Crystal Stacker is freeware. This means you can pass copies around freely provided you include this
```

Comment:
Personally I would put all of the text within body tags, as none of it specifically looks like is it title or optional. Pull request submitted.